src/meta/src/hummock/compactor_manager.rs

```diff
@@ -240,7 +240,10 @@ impl CompactorManager {
         self.task_heartbeats.write().remove(&context_id).is_some()
     }
 
-    pub fn get_expired_tasks(&self) -> Vec<(HummockContextId, CompactTask)> {
+    pub fn get_expired_tasks(
```

Comment:
It makes a little confused to me.

`get_expired_tasks` try to traverse the heartbeats info and check if the task expired due to heartbeat timeout.

IMO, `split_cancel` is another invalid type that need manual cancel, but its not belong to the TaskHeartbeat timeout expire

Reply:
Why not just call the original `get_expired_tasks` and merge them with `split_cancel`? `get_expired_tasks` are only meant to return the tasks that have no progress according to the heartbeat. So I guess it's better to rename it as `get_heartbeat_expired_tasks`.

Reply:
> It makes a little confused to me.
> 
> `get_expired_tasks` try to traverse the heartbeats info and check if the task expired due to heartbeat timeout.
> 
> IMO, `split_cancel` is another invalid type that need manual cancel, but its not belong to the TaskHeartbeat timeout expire

fixed

Reply:
> Why not just call the original `get_expired_tasks` and merge them with `split_cancel`? `get_expired_tasks` are only meant to return the tasks that have no progress according to the heartbeat. So I guess it's better to rename it as `get_heartbeat_expired_tasks`.

fixed